samples/gramine-hello/README.md

```diff
@@ -42,5 +42,5 @@ curl -k --data-binary @manifest.json https://localhost:4433/manifest
 The type of the Marble is defined in the `manifest.json`. In this example, the manifest defines a single Marble, which is called "hello". To run the application, you need to set the `EDG_MARBLE_TYPE` environement variable to that name.
```

Comment:
Can you actually override an environment variable defined inside the `Makefile` from the outside by e.g.:
`EDG_MARBLE_TYPE=myownmarble make run`
?

If no, you may want to adjust this sentence to highlight that the definition is in the `Makefile`.

Reply:
Maybe we should just remove that sentence. Does not make a lot of sense if we already set default values